ansible-roles/k8s_fabric_peer/templates/configmap.yaml.j2

```diff
@@ -551,14 +551,24 @@ data:
 
         # List of directories to treat as external builders and launchers for
         # chaincode. The external builder detection processing will iterate over the
-        # builders in the order specified below.
-        {% if k8s_fabric_peer_chaincode_external %}
+        # builders in the order specified below. FNORD
```

Comment:
FNORD? heh

Reply:
lol i don't know how that ended up there